### ISSUE TITLE
[Bug] Unable to get downloads

### ISSUE THREAD

**Issue by aliglelo** [subhra74/xdm#197]
**XDM does not catches any downloads**
I am using firefox 76.0.1 and using [GhackUser.js](https://github.com/ghacksuserjs/ghacks-user.js/) as a custom user.js for privacy controls but the problem is that after implementing that User.js , XDM is unable to catch any downloads from firefox.So can there be any fix in user.js that i can make to get xdm working again.Since , i have already tried to reinstall the extension and tried restarting the browser etc.

**To Reproduce**
1. Go to 'any downloading website'
2. Click on 'download'
3. If browser monitoring is enabled by extension , nothing happen(no download or anything)

**Expected behavior**
XDM should get the download

**please complete the following information:**
 - OS: Linux
 - Browser: Firefox
 - XDM addon Version: Latest firefox addon store version
 - XDM Version: 7.2.11

**Comment by subhra74** [comment]
Please go to firefox addon page, open the console using below steps and try to download something. Any errors should be displayed in console

![image](https://user-images.githubusercontent.com/8403090/83196791-c137f100-a13c-11ea-989f-31b24f95f3aa.png)

![image](https://user-images.githubusercontent.com/8403090/83196835-d6148480-a13c-11ea-97c4-6a6d419a1b63.png)

![image](https://user-images.githubusercontent.com/8403090/83196895-ef1d3580-a13c-11ea-9349-02899c66be2d.png)

**Comment by aliglelo** [comment]
Okay so here is how I solved the problem.
First of all I inspected the console in inspect mode of XDM firefox extension and noticed that `First party Isolation` was a problem as a security measure `ghackuser.js` was enabling that Isolation.

- After some research I found an article [here](https://www.ctrl.blog/entry/firefox-fpi.html) about the feature and found that it restricts the Extension APIs too.

- After manunally setting the option to disable everything started to work perfectly.
